source/stylesheets/_tab-code_signing.scss.erb

```diff
@@ -450,9 +450,19 @@
 									display: none;
 								}
 
-								svg {
+								.opened-eye, .closed-eye {
```

Comment:
We should rename these to `eye` and `eye_crossed` - the CSS classes (`eye-crossed` here) and the .svg filenames as well, for clarity.